core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSequencerReference.java

```diff
@@ -113,14 +170,28 @@ public boolean isDeletable() {
 
   @Override
   public String toString() {
-    StringBuilder sb = new StringBuilder();
-    sb.append(getId());
-    sb.append(" : ");
-    sb.append(getName());
-    sb.append(" : ");
-    sb.append(getFQDN());
-    sb.append(" : ");
-    sb.append(getAvailable());
-    return sb.toString();
+    return "AbstractSequencerReference [id=" + id + ", name=" + name
+        + ", platform=" + (platform == null ? null : platform.getPlatformId()) + ", available=" + available
+        + ", ip=" + (ip == null ? null : ip.toString())
+        + ", serialNumber=" + serialNumber
+        + ", dateCommissioned=" + (dateCommissioned == null ? null : dateCommissioned.toString())
+        + ", dateDecommissioned=" + (dateDecommissioned == null ? null : dateDecommissioned.toString())
+        + ", upgradedSequencerReference=" + (upgradedSequencerReference == null ? null : upgradedSequencerReference.getId()) + "]";
```

Comment:
If you'd like to get the String value of something and want it to be null friendly you could always try this:
```java
    String s = String.valueOf(obj);
```
I believe `String.valueOf(obj)` looks like this under the hood:

```java
public static String valueOf(Object obj) {
    return (obj == null) ? "null" : obj.toString();
}
```

Applies file wide :neckbeard: #notadealbreaker